### DISCUSSION TITLE
Automation for access key update

### DISCUSSION THREAD

**Discussion by taprackbang09** [getredash/redash#5944] [Q&A] [answered]
Hi there,
Due to the security policies in our company, we need to rotate keys for Athena every ninety days. I am wondering if we can automate the process for updating these keys in Redash. Could some scripts be created for similar cases?
![image](https://user-images.githubusercontent.com/1760612/227902885-8f822787-da7f-49ab-8f54-e7c477140553.png)

**Comment by arikfr** [comment, marked as answer]
Yes this is possible using the API. If you want to automate this using Python you can use the [toolbelt](https://github.com/getredash/redash-toolbelt) or [redash-python](https://github.com/Blacksuan19/redash-python).

**Reply by taprackbang09** [reply]
Thanks!
